src/main/java/com/example/petbuddybackend/service/chat/ChatService.java

```diff
@@ -79,15 +78,17 @@ public ChatMessageDTO createMessage(Long chatId, String principalEmail, ChatMess
                 chatRoom.getClient().getAccountData() :
                 chatRoom.getCaretaker().getAccountData();
 
-        return chatMapper.mapToChatMessageDTO(createMessage(chatRoom, sender, chatMessage.getContent()));
+        return chatMapper.mapToChatMessageDTO(createMessageConvertTimeZone(chatRoom, sender, chatMessage.getContent()));
     }
-
     @Transactional
-    public ChatMessageDTO createMessage(Long chatId, String principalEmail, ChatMessageSent chatMessage, Role role, ZoneId timeZone) {
-        return convertTimeZone(
-                createMessage(chatId, principalEmail, chatMessage, role),
-                timeZone
-        );
+    public ChatMessageDTO createMessageConvertTimeZone(
+            Long chatId,
+            String principalEmail,
+            ChatMessageSent chatMessage,
+            Role role,
+            ZoneId timeZone
+    ) {
+        return chatMapper.mapTimeZone(createMessage(chatId, principalEmail, chatMessage, role), timeZone);
     }
 
     public ChatMessageDTO createChatRoomWithMessage(
```

Comment:
a tu nie zostawiamy tylko tej metody, ktora przyjmuje ZoneId i jak nie bedzie podane to zmapuje na default?